plugins/inputs/win_perf_counters/README.md

```diff
@@ -92,6 +104,14 @@ This must follow before other plugin configurations,
 beneath the main win_perf_counters entry, `[[inputs.win_perf_counters]]`.
 
 Following this are 3 required key/value pairs and three optional parameters and their usage.
+#### Computers
+*Optional*
+
+Host names or IPs of computers to gather counters from. User, under which Telegraf runs, must be already authenticated to the remote computer(s). E.g. via Windows sharing `net use \\SQL-SERVER-01`.
+Skip, if gather only from localhost.
```

Comment:
For config consistency, is it valid to show that `computers = ["localhost"]` is the default?
Also, are IPs valid?

Reply:
I think the default should be shown with `localhost`, this gives a nice progression when removing the item.  Internally it should be treated the same.  This will work better and be more important when we update the plugin to match the current Telegraf styling.

This looks like we are only adding a server:
```diff
- computer = []
+ computer = ["SQL-SERVER-01"]
```

This style appears more accurate, we are removing localhost and adding another computer:
```diff
- computer = ["localhost"]
+ computer = ["SQL-SERVER-01"]
```

Reply:
Ok.